app/styles/main.scss

```diff
@@ -68,3 +68,7 @@ body {
 }
 
 .qtip-wide{max-width:380px !important;}
+
+#range-chart g.y{  /* hides the y axis on the rangeChart of the line chart*/
```

Comment:
This style should apply to all potential line-charts. Using # will limit the usage